ui/src/app/base/components/FormikFormButtons/FormikFormButtons.tsx

```diff
@@ -115,7 +125,9 @@ export const FormikFormButtons = <V,>({
               })}
               data-test="cancel-action"
               disabled={cancelDisabled}
-              onClick={onCancel ? () => onCancel(formikContext) : undefined}
+              onClick={
+                onCancel ? () => onCancel(values, formikContext) : undefined
```

Comment:
I can't remember if there was a reason why these were set to `undefined`, but does it work when they're `null` instead?

Reply:
Ahh looks like it's because the `Button` component doesn't accept `null`. I'll file an issue there